src/Curl/MultiCurl.php

```diff
@@ -190,12 +190,16 @@ public function addPatch($url, $data = array())
      * @access public
      * @param  $url
      * @param  $data
+     * @param  $post_redirect_get If true, will cause 303 redirections to be followed using
```

Comment:
I would trigger this just to output the response by default so the user can
deal with the return codes themselves
On Jan 31, 2016 11:32 AM, "Zach Borboa" <notifications@github.com> wrote:

> In src/Curl/MultiCurl.php
> <https://github.com/php-curl-class/php-curl-class/pull/247#discussion_r51365778>
> :
>
> > @@ -190,12 +190,16 @@ public function addPatch($url, $data = array())
> >       * @access public
> >       * @param  $url
> >       * @param  $data
> > +     * @param  $post_redirect_get If true, will cause 303 redirections to be followed using
>
> Also include 301, 302, and 303 here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/php-curl-class/php-curl-class/pull/247/files#r51365778>
> .
>